jsonnet/lib/token-refresher.libsonnet

```diff
@@ -0,0 +1,126 @@
+// These are the defaults for this components configuration.
+// When calling the function to generate the component's manifest,
+// you can pass an object structured like the default to overwrite default values.
+local defaults = {
+  local defaults = self,
+  name: error 'must provide name',
+  image: 'quay.io/observatorium/token-refresher',
+  version: error 'must provide version',
+  namespace: error 'must provide namespace',
+  url: error 'must provide target url',
+  oidcSecret: {
```

Comment:
Rather than making a nested object here, i think we could be consistent with Observatorium code and do:
```
secretName:...
issuerURLKey:...
clientSecretKey:...
... etc
```
Wdyt?

Reply:
Done.